src/bantz/google/consent_wizard.py

```
@@ -0,0 +1,566 @@
+    def _disconnect_service(self, svc: ServiceDefinition) -> bool:
+        """Disconnect (revoke) a service by deleting its token."""
+        try:
+            token_path_str = os.getenv(svc.token_path_key) or svc.default_token_path
+            token_path = Path(os.path.expanduser(token_path_str)).resolve()
+            if token_path.exists():
+                token_path.unlink()
+                print(f"  {_C.YELLOW}Token silindi: {token_path}{_C.RESET}")
+
+            self._statuses[svc.id] = ServiceStatus(service_id=svc.id)
+            _save_service_status(self._statuses)
+            print(f"  {_C.GREEN}✓ {svc.name} bağlantısı kaldırıldı.{_C.RESET}")
```

Ruff (0.15.0)

[warning] String contains ambiguous `ı` (LATIN SMALL LETTER DOTLESS I). Did you mean `i` (LATIN SMALL LETTER I)?

(RUF001)